test/functional/api/cas/cache.py

```diff
@@ -68,6 +68,10 @@ def get_status(self):
         status = self.get_cache_statistics()["status"].replace(' ', '_')
         return CacheStatus[status]
 
+    @property
+    def size(self):
+        return self.get_cache_statistics()["cache size"]
```

Comment:
If pull request #218 gets merged this line should look like this:
`return self.get_cache_statistics().config_stats.cache_size`

Reply:
Done